src/qlipperpreferencesdialog.ui

```diff
@@ -35,44 +35,13 @@
         <x>0</x>
         <y>0</y>
         <width>340</width>
-        <height>217</height>
+        <height>302</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Preferences</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
-       <item row="4" column="0" colspan="2">
```

Comment:
> This was my first PR and literally last one.

What's the problem here? You don't seem like a git novice. If this is a realy annoying task for you, I (we) can handle it in behalf of you. But you should say it.... and not just close the PR in anger.